src/fabric/src/fabric_util.erl

```diff
@@ -302,7 +304,17 @@ is_users_db(DbName) ->
 path_ends_with(Path, Suffix) ->
     Suffix =:= couch_db:dbname_suffix(Path).
 
-fake_db(DbName, Opts) ->
+make_cluster_db(#shard{dbname = DbName, opts = Options}) ->
```

Comment:
The name is confusing. `make` is too close to `create` in meaning.

Reply:
You mean that it could be confused with `fabric:create_db/1` in that it might be actually creating shard maps and shard files on disk instead of creating a database handle? How about `open_clustered_db`?

Reply:
Fixed with open_clustered_db.